src/plugins/wb-onpage-submit/wb-onpage-submit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @title WET-BOEW wb-onpage-submit
+ * @overview This plugin implements AJAX request for form data
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @masterbee, @namjohn920
+ **/
+
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-onpage-submit",
+	selector = ".provisional." + componentName,
+	initEvent = "wb-init" + selector,
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector );
+
+		elm.addEventListener( "submit", function( e ) {
+			e.preventDefault();
+			var $elm = $( this );
+			try {
+				$.ajax( {
+					type: $elm.attr( "method" ),
+					url: $elm.attr( "action" ),
+					data: $elm.serialize() // serializes the form's elements.
+				} );
+			} catch ( error ) {
+				$( this.querySelector( ".formContents" ) ).toggleClass( "hidden" );
+				$( this.querySelector( ".failureMessage" ) ).toggleClass( "hidden" );
+				throw error;
+			}
+			$( this.querySelector( ".formContents" ) ).toggleClass( "hidden" );
+			$( this.querySelector( ".successMessage" ) ).toggleClass( "hidden" );
```

Comment:
use a Jquery selector or a CSS selector, no need to swapping between both JQuery and VanillaJS